src/api/EditCatProfile.js

```diff
@@ -0,0 +1,41 @@
+// Funktion um ein verändertes Katzenprofil zu speichern
+import { useMutation } from "@apollo/client";
+
+import { CatProfileMutation } from "./constants/EditCatProfileMutation";
+
+export default function EditCatProfile(props) {
+  const input = {};
+  if (props.id !== null) {
+    input.id = props.id;
```

Comment:
warum so umständlich? wir können einfach ein cat objekt mit allen werten übergeben. Wenn ein Wert ist wie in der db -> keine änderung
und props ist hier glaube ich kein guter parameter name, da es keine react component ist